platform/darwin/scripts/generate-style-code.js

```diff
@@ -107,9 +107,9 @@ global.propertyReqs = function (property, layoutPropertiesByName, type) {
             return '`' + camelizeWithLeadingLowercase(req['!']) + '` is set to `nil`';
         } else {
             let name = Object.keys(req)[0];
-            return '`' + camelizeWithLeadingLowercase(name) + '` is set to ' + describeValue(req[Object.keys(req)[0]], layoutPropertiesByName[name], type);
+            return '`' + camelizeWithLeadingLowercase(name) + '` is set to ' + describeValue(req[name], layoutPropertiesByName[name], type);
         }
-    }).join(', and ') + '. Otherwise, it is ignored.';
+    }).join('; and ') + '. Otherwise, it is ignored.';
```

Comment:
Oh, a semicolon is definitely helpful if a requirement does include a comma, though. It isn’t wrong either way, since you’ve simplified the verbiage a bit, just a stylistic choice.